dist/uFuzzy.d.ts

```diff
@@ -5,8 +5,8 @@ declare class uFuzzy {
 	search(
 		haystack: string[],
 		needle: string,
-		outOfOrder = false,
-		infoThresh = 1e3,
+		outOfOrder?: boolean,
+		infoThresh?: number,
```

Comment:
maybe keep them as comments?

```suggestion
		outOfOrder?: boolean, // false
		infoThresh?: number,  // 1e3
```

i wish TS had a way to indicate defaults for optional args :(

Reply:
Yeah good point. I'll add them in doc comments so they show up in intellisense on hover 👍